### DISCUSSION TITLE
👋Welcome to Beta Discussions!

### DISCUSSION THREAD

**Discussion by N1kyyyy** [N1kyyyy/Beta#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by N1kyyyy** [comment]
Just a Beta

**Reply by 5SSjw** [reply]
Just a Beta

**Comment by 5SSjw** [comment]
Just a Beta

**Comment by 5SSjw** [comment]
Just a Beta
